python/lsst/obs/base/instrument.py

```diff
@@ -141,6 +179,123 @@ def applyConfigOverrides(self, name, config):
             if os.path.exists(path):
                 config.load(path)
 
+    def writeCameraGeom(self, butler):
+        """Write the default camera geometry to the butler repository
+        with an infinite validity range.
+
+        Expected to be called from an instrument subclass
+        ``writeCuratedCalibrations`` implementation.
```

Comment:
I had wondered about that since I wasn't sure what was driving it to be abstract. I can use a default that calls those two things and see how obs_subaru gets on with that.